producers/scalefactors.py

```diff
@@ -356,3 +356,19 @@
     output=[q.btag_weight],
     scopes=["tt", "mt", "et", "mm", "em", "ee"],
 )
+
+MuonIDSF_friends_1 = Producer(
+    name="MuonIDSF_friends_1",
+    call='scalefactor::embedding::muon_sf({df}, {input}, {output}, "{muon_sf_file}", "emb", "{muon_id_sf}")',
+    input=[q.pt_1, q.eta_1],
+    output=[q.id_wgt_mu_friend_1],
+    scopes=["mt", "et"],
+)
+
+MuonIsoSF_friends_1 = Producer(
+    name="MuonIsoSF_friends_1",
+    call='scalefactor::embedding::muon_sf({df}, {input}, {output}, "{muon_sf_file}", "emb", "{muon_iso_sf}")',
+    input=[q.pt_1, q.eta_1],
+    output=[q.iso_wgt_mu_friend_1],
+    scopes=["mt", "et"],
+)
```

Comment:
I thought these are only for testing. Normally these SFs would already be calculated in the ntuple, right?
If yes, I would not add them since we have other examples for friends like fastmtt.